### PR TITLE
publish-barshare-gbfs: add index/overview page

### PR DESCRIPTION
**Pull request by derhuerst**
missing:

- [ ] ~~generate & link to schema.org `Dataset` #2~~ – will be done in another PR
- [ ] ~~generate & link to DataPage #2~~ – will be done in another PR
- [x] move combined feed into `all`?

### CODE REVIEW

File: publish-barshare-gbfs/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	</ul>
+	<h3>Feed mit Leihfahrrädern</h3>
+	<ul>
+		<li><a href="./bicycle/gbfs.json"><code>gbfs.json</code></a> (<a href="https://creativecommons.org/publicdomain/zero/1.0/"><code>CC0-1.0</code></a>)</li>
+	</ul>
+	<h3>Feed mit Leihfahrautos</h3>
+	<ul>
+		<li><a href="./car/gbfs.json"><code>gbfs.json</code></a> (<a href="https://creativecommons.org/publicdomain/zero/1.0/"><code>CC0-1.0</code></a>)</li>
+	</ul>
+	<h3>Feed mit anderen Fahrzeugen</h3>
```
Comment (hbruch) [inline publish-barshare-gbfs/index.html:74]:
```suggestion
	<h3>Feed mit sonstigen Fahrzeugen (derzeit insbesondere Lastenräder, für die in GBFS 2.2 noch kein form_factor definiert ist.)</h3>
```

File: publish-barshare-gbfs/index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	<p><a href="https://github.com/NABSA/gbfs/blob/v2.2/README.md" rel="noreferrer noopener">Mehr Informationen zum GBFS-Format (Englisch)</a></p>
+	<h3>Feed mit <em>allen</em> Fahrzeugen</h3>
+	<ul>
+		<li><a href="./gbfs.json"><code>gbfs.json</code></a> (<a href="https://creativecommons.org/publicdomain/zero/1.0/"><code>CC0-1.0</code></a>)</li>
+	</ul>
+	<h3>Feed mit Leihfahrrädern</h3>
+	<ul>
+		<li><a href="./bicycle/gbfs.json"><code>gbfs.json</code></a> (<a href="https://creativecommons.org/publicdomain/zero/1.0/"><code>CC0-1.0</code></a>)</li>
+	</ul>
+	<h3>Feed mit Leihfahrautos</h3>
```
Comment (hbruch) [inline publish-barshare-gbfs/index.html:70]:
```suggestion
	<h3>Feed mit Carsharing-Fahrzeugen</h3>
```

**Review by hbruch — COMMENTED** [review]
Ich würde noch zwei kleinere Formulierungen ändern

File: publish-barshare-gbfs/index.html (hunk 1 above)
Comment (derhuerst) [reply publish-barshare-gbfs/index.html:74]: @leonardehrenfried See #4.

**Pull request merged**